internal/raft/node.go

```diff
@@ -321,6 +321,10 @@ func (rc *raftNode) startRaft() {
 		}
 	}
 	rc.snapshotter = snap.New(rc.snapdir)
+	var err error
+	if rc.raftStorage, err = storage.NewTempEntryStore(rc.id); err != nil {
```

Comment:
Probably we might want to offer a configuration to provide the directory? In certain clouds it might have a small tmp disk

Reply:
Ack. Fixing this point.